docker-compose.yml

```diff
@@ -3,19 +3,12 @@ services:
   editor:
     build:
       context: .
+      args:
+        SPOOF_SINOPIA_SERVER: 'false'
+        TRELLIS_BASE_URL: http://platform:8080
     ports:
       - 8000:8000
     depends_on:
-      - pipeline
-  pipeline:
```

Comment:
Remove pipeline service from editor docker-compose config bc not currently used for anything.